mapie/classification.py

```diff
@@ -292,9 +303,18 @@ def _check_cv(
         """
         if cv is None:
             return "prefit"
-        if cv == "prefit":
+        if isinstance(cv, int):
+            if cv == -1:
+                return LeaveOneOut()
+            if cv >= 2:
+                return KFold(n_splits=cv)
+        if isinstance(cv, BaseCrossValidator) or (cv == "prefit"):
             return cv
-        raise ValueError("Invalid cv argument." "Allowed value is 'prefit'.")
+        raise ValueError(
+            "Invalid cv argument. "
+            "Allowed values are None, -1, int >= 2, 'prefit', "
+            "or a BaseCrossValidator object (Kfold, LeaveOneOut)."
```

Comment:
Can we align this message with that of `MapieRegressor` ?

Reply:
See above.

Reply:
Function updated and moved in `utils.py`.